src/Microsoft.Diagnostics.ExtensionCommands/GCRootCommand.cs

```diff
@@ -108,6 +109,30 @@ public override void Invoke()
             }
 
             Console.WriteLine($"Found {count:n0} unique roots.");
```

Comment:
If you use Trace.TraceInformation is will go into the separate *.soslog files and not interfere with the regex's.

Reply:
Ah, I was thinking the regex would still work because I only tacked on output to the end.

Good idea though, I'll push a Trace.TraceInformation change after CI finishes processing this run.